.github/workflows/releaser.yml

```
@@ -0,0 +1,44 @@
+name: Library Releaser
+
+on:
+  push:
+    tags:
+      - "*"
```

When we make a version tag we start with a `v` so can we change this to:
```suggestion
      - "v*"
```